tools/build/pack_cordova_sample.py

```diff
@@ -298,6 +298,18 @@ def replaceKey(file_path, content, key):
         return False
     return True
 
+def createIndexFile(index_file_path=None, hosted_app=None):
+    try:
+        if hosted_app == "statusbar":
+            html_content = '<script src="./cordova.js"></script>\n<div id="header">\n  <h3 id="main_page_title">Status Bar Test In Fullscreen</h3>\n</div>\n<br><br>\n<p>Click "Status Bar Show" button to show status bar:</p>\n<button onclick="StatusBar.show();">Status Bar Show</button><br><br>\n<p>Click "Status Bar Hide" button to hide status bar:</p>\n<button onclick="StatusBar.hide();">Status Bar Hide</button>'
```

Comment:
Not a good coding style to write a so long string in one line. Please divide into multiple lines.